l10n_nl_xaf_auditfile_export/controllers/main.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import http
+from openerp.http import request
+from openerp.addons.web.controllers.main import content_disposition
+
+from ..models.xaf_auditfile_export import get_auditfile_path
+
+
+class Main(http.Controller):
+
+    @http.route('/file_cabinet/auditfile', type='http', auth='user')
```

Comment:
This strikes me as audiosyncratic. Also, seems like the auditfile can be downloaded by any user if they know the url?